src/logging/console.lua

```diff
@@ -10,14 +10,14 @@
 local logging = require"logging"
 
 function logging.console(params, ...)
-  params = logging.getDeprecatedParams({ "logPattern" }, params, ...)
+  params = logging.getDeprecatedParams({ "logPattern","levels" }, params, ...)
```

Comment:
please use a space after a comma, this has multiple occurences (this entire file is not consistent, but new code should probably be, thx!)